modules/recommended-scale-practices.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-cluster-scaling-practices.adoc
+
+[id="recommended-scale-practices_{context}"]
+= Recommended practices for scaling the cluster
+
+When scaling up the cluster to higher node counts, you should spread the nodes
+across all of the available zones for higher availability. If the replicas in
+the machinesets are set to higher numbers all at one time, the controller might
+have issues creating the machines. The number of requests the cloud platform,
+which {product-title} is deployed on top of, is able to handle impacts the
+process. The controller will start to query more while trying to create, check,
```

Comment:
"The controller will start to query more while trying to create, check, and update the machines with the status."

Can we be a little clearer about why this is a problem? Do the excessive queries have the potential to get your cluster into state that it will have trouble recovering from?

Reply:
The cloud platform on which openshift is deployed has API request limits and excessive queries might lead to machine creation failures due to cloud platform limitations.